src/test/scala-2.11/markets/participants/OrderCancelerSpec.scala

```diff
@@ -15,21 +15,21 @@ limitations under the License.
 */
 package markets.participants
 
-import akka.actor.ActorSystem
+import akka.actor.{ActorRef, ActorSystem}
 import akka.agent.Agent
 import akka.testkit.{TestProbe, TestActorRef, TestKit}
 
 import java.util.UUID
 
-import markets.Cancel
+import markets.{Cancel, Canceled}
 import markets.orders.limit.LimitAskOrder
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Imports should be sorted alphabetically](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1705065396/issues/source?bid=2921361&fileBranchId=3046267#l25)